src/data_storage/metatype_key_storage.ts

```diff
@@ -124,14 +124,36 @@ export default class MetatypeKeyStorage extends PostgresStorage{
         let i = 1;
 
         Object.keys(updatedField).map(k => {
-            updateStatement.push(`${k} = $${i}`);
-            values.push(updatedField[k]);
-            i++
+           if(k === `created_at` || k === `created_by` || k === 'modified_at' || k === 'modified_by') {
+               return
+           }
+
+           // we must stringify the default value as it could be something other
+           // than a string, we store it as a string in the DB for ease of use
+           if(k === 'default_value') {
```

Comment:
would it be easier for this to be `k === 'default_value' || k === 'options'`?

Reply:
🤦 yes, yes it would. I'll fix it